test/renderToJSON.spec.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { expect } from 'chai';
+
+import * as tags from '../src';
+
+const renderToJSON = tags.renderToJSON;
+
+describe('render to JSON', () => {
+  it('should render simple <Mjml/> tag with content', () => {
+    expect(renderToJSON(<tags.Mjml>Content</tags.Mjml>)).to.eql({
+      tagName: 'mjml',
+      attributes: {},
+      content: 'Content',
+    });
+  });
+
+  it('should render <Mjml/> example with children', () => {
+    expect(
+      renderToJSON(
+        <tags.Mjml>
+          <tags.MjmlBody>
+            <tags.MjmlSection>
+              <tags.MjmlColumn>
+                <tags.MjmlImage
+                  width="100px"
+                  src="/assets/img/logo-small.png"
+                />
+                <tags.MjmlDivider borderColor="#F45E43" />
+                <tags.MjmlText
+                  fontSize="20px"
+                  color="#F45E43"
+                  fontFamily="helvetica"
+                >
+                  Hello World
```

Comment:
Please add some html with `<>&` chars inside, just to make sure they are properly escaped.